tooling/log_analysis.ipynb

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
nit: can you move this to a subfolder? We generally have `tooling/{tool}/{source_code}`